app/models/manageiq/providers/redhat/infra_manager/ovirt_services/strategies/v3.rb

```diff
@@ -190,6 +190,16 @@ def vm_reconfigure(vm, options = {})
       _log.info("#{log_header} Completed.")
     end
 
+    def advertised_images
+      ext_management_system.with_provider_connection do |rhevm|
+        rhevm.iso_images.collect { |image| image[:name] }
+      end
+    rescue Ovirt::Error => err
+      name = ext_management_system.try(:name)
+      _log.error("Error Getting ISO Images on ISO Datastore on Management System <#{name}>: #{err.class.name}: #{err}")
+      raise ManageIQ::Providers::Redhat::InfraManager::OvirtServices::Error err
```

Comment:
I had  ManageIQ::Providers::Redhat::InfraManager::OvirtServices::Error.new err
which works but rubocop did not like it.... any way fixed it, tnx.